crates/hotshot/src/traits/networking/web_server_network.rs

```diff
@@ -503,47 +503,38 @@ impl<TYPES: NodeType> Inner<TYPES> {
                 }
             }
 
-            let maybe_event = receiver.try_recv();
-            match maybe_event {
-                Ok(event) => {
-                    match event {
-                        // TODO ED Should add extra error checking here to make sure we are intending to cancel a task
-                        ConsensusIntentEvent::CancelPollForVotes(event_view)
-                        | ConsensusIntentEvent::CancelPollForProposal(event_view)
-                        | ConsensusIntentEvent::CancelPollForDAC(event_view)
-                        | ConsensusIntentEvent::CancelPollForViewSyncCertificate(event_view)
-                        | ConsensusIntentEvent::CancelPollForVIDDisperse(event_view)
-                        | ConsensusIntentEvent::CancelPollForLatestProposal(event_view)
-                        | ConsensusIntentEvent::CancelPollForLatestViewSyncCertificate(
-                            event_view,
-                        )
-                        | ConsensusIntentEvent::CancelPollForViewSyncVotes(event_view) => {
-                            if view_number == event_view {
-                                debug!("Shutting down polling task for view {}", event_view);
-                                return Ok(());
-                            }
+            if let Ok(event) = receiver.try_recv() {
```

Comment:
Most of this is a clippy-mandated change to delete the `match` after removing the log on `Err(_)`. The logic should be exactly the same.